### ISSUE TITLE
Running hp_search.py takes me 2 days and get the best eao=0.383. Can anyone get 0.41???

### ISSUE THREAD

**Issue by alyssa121** [STVIR/pysot#242]
I use the config of "siamrpn_r50_l234_dwxcorr" and set batchsize to 28.  It takes me 3 days(almost 72 hours) to train with 2 2080ti ,and i just get the best eao=0.355 on epoch 18. I run hp_search.py with epoch13 and still can't achieve 0.41. Could anybody who achieve 0.41 tells me your hyperparameter and batchsize and num of gpus?

Here is the result after hp_search.py:
------------------------------------------------------------------------------------------------
|                  Tracker Name                  | Accuracy | Robustness | Lost Number |  EAO  |
------------------------------------------------------------------------------------------------
| checkpoint_e13_r255_pk-0.050_wi-0.300_lr-0.400 |  0.599   |   0.239    |    51.0     | 0.383 |
| checkpoint_e13_r255_pk-0.200_wi-0.350_lr-0.450 |  0.592   |   0.239    |    51.0     | 0.379 |
| checkpoint_e13_r255_pk-0.150_wi-0.350_lr-0.500 |  0.595   |   0.248    |    53.0     | 0.378 |
| checkpoint_e13_r255_pk-0.050_wi-0.350_lr-0.500 |  0.600   |   0.253    |    54.0     | 0.377 |
| checkpoint_e13_r255_pk-0.150_wi-0.300_lr-0.500 |  0.595   |   0.253    |    54.0     | 0.376 |
| checkpoint_e13_r255_pk-0.250_wi-0.350_lr-0.400 |  0.590   |   0.267    |    57.0     | 0.376 |
| checkpoint_e13_r255_pk-0.350_wi-0.300_lr-0.400 |  0.589   |   0.267    |    57.0     | 0.375 |
| checkpoint_e13_r255_pk-0.150_wi-0.250_lr-0.450 |  0.595   |   0.248    |    53.0     | 0.373 |
| checkpoint_e13_r255_pk-0.150_wi-0.250_lr-0.350 |  0.593   |   0.258    |    55.0     | 0.372 |
| checkpoint_e13_r255_pk-0.350_wi-0.400_lr-0.400 |  0.596   |   0.276    |    59.0     | 0.372 |
| checkpoint_e13_r255_pk-0.200_wi-0.350_lr-0.400 |  0.594   |   0.262    |    56.0     | 0.372 |
| checkpoint_e13_r255_pk-0.300_wi-0.350_lr-0.500 |  0.596   |   0.267    |    57.0     | 0.371 |
| checkpoint_e13_r255_pk-0.050_wi-0.300_lr-0.450 |  0.602   |   0.267    |    57.0     | 0.370 |
| checkpoint_e13_r255_pk-0.200_wi-0.400_lr-0.350 |  0.594   |   0.272    |    58.0     | 0.370 |
| checkpoint_e13_r255_pk-0.050_wi-0.350_lr-0.400 |  0.601   |   0.248    |    53.0     | 0.369 |
| checkpoint_e13_r255_pk-0.100_wi-0.300_lr-0.400 |  0.600   |   0.272    |    58.0     | 0.369 |
| checkpoint_e13_r255_pk-0.050_wi-0.250_lr-0.400 |  0.603   |   0.281    |    60.0     | 0.369 |
| checkpoint_e13_r255_pk-0.100_wi-0.350_lr-0.500 |  0.598   |   0.290    |    62.0     | 0.366 |
| checkpoint_e13_r255_pk-0.350_wi-0.350_lr-0.500 |  0.598   |   0.281    |    60.0     | 0.366 |
| checkpoint_e13_r255_pk-0.050_wi-0.300_lr-0.350 |  0.601   |   0.262    |    56.0     | 0.366 |
------------------------------------------------------------------------------------------------

**Comment by yc-zc** [comment]
https://github.com/STVIR/pysot/issues/94
